.travis.yml

```diff
@@ -1,15 +1,38 @@
 # Config file for automatic testing at travis-ci.org
 language: python
-python:
-  - "3.5"
-env:
-  - TOX_ENV=py27-django18
-  - TOX_ENV=py35-django18
-  - TOX_ENV=py27-django19
-  - TOX_ENV=py35-django19
-  - TOX_ENV=py27-django110
-  - TOX_ENV=py35-django110
+
+matrix:
+    include:
+      # Python version is just ofr the look on travis.
+        - python: 2.7
+          env: TOX_ENV=py27-django18
+
+        - python: 2.7
+          env: TOX_ENV=py27-django19
+
+        - python: 2.7
+          env: TOX_ENV=py27-django110
+
+        - python: 3.5
+          env: TOX_ENV=py35-django18
+
+        - python: 3.5
+          env: TOX_ENV=py35-django19
+
+        - python: 3.5
+          env: TOX_ENV=py35-django110
+
+        - env: TOX_ENV=linters
+
+
+before_install:
+  - pip install codecov
+
 install:
   - pip install tox
+
 script:
   - tox -e $TOX_ENV
+
+after_success:
+  - codecov
```

Comment:
(there's probably a last bit of work to do so that codecov reports correctly the name of the build)